modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue

```diff
@@ -41,13 +41,13 @@
           <li class="apos-area-widget__breadcrumb" data-apos-widget-breadcrumb="0">
             <AposButton
               type="quiet"
-              @click="foreign ? $emit('edit', i) : false"
+              @click="foreign ? $emit('edit', i) : null"
+              @dblclick.native="(!foreign && !isContextual) ? $emit('edit', i) : null"
```

Comment:
There is a lint error here based on this [breaking change](https://v3-migration.vuejs.org/breaking-changes/v-on-native-modifier-removed.html#v-on-native-modifier-removed).
Not sure how to replace it though but it would be better (since we will need to migrate to Vue 3 soon). If you can't I guess we need to modify the eslint config.

Reply:
We should override this Vue 3 - specific rule in our linter config, rather than making Stu change correct Vue 2 code. This is the only way to do this in Vue 2.

We may need to fork the Vue linter rules module and maintain a Vue 2 - compatible set ourselves (although in practice, there's little reason to change them, so the "maintenance" would not be difficult).